### ISSUE TITLE
global variable

### ISSUE THREAD

**Issue by prathibhacdac** [InnovateAsterisk/Browser-Phone#241]
I've declared the following global variable in phone.js

let did = new Array();
let i = 0;

Inside ConferenceDail() function I've used:

 did[i++] = session.remoteIdentity.uri.user;

Everytime the ConferenceDail function is called, i is always 0. I'm expecting i's value to be incremented each time the ConferenceDail function is called. Why is it not happening?

**Comment by InnovateAsterisk** [comment]
The ```ConferenceDail()``` function is in the middle of a sequence of events that needs to occur when conferencing in another call. Conference calls implement a child session, attached to the parent session.

Can you explain what you are trying to achieve?

**Comment by prathibhacdac** [comment]
I'm trying to get all the parent ids.

**Comment by InnovateAsterisk** [comment]
To do what? explain further. 

Are you perhaps not looking for the Asterisk ConfBridge solution?

**Comment by prathibhacdac** [comment]
Does the ConfBridge work with Browser phone?

**Comment by InnovateAsterisk** [comment]
Yes, its mostly implemented, I still have to complete the inter-conference message sending, that a caller can send a text message to all participants of the conference call.

Make sure you implement something like this: 
https://github.com/InnovateAsterisk/Browser-Phone/blob/master/config/confbridge.conf

You can use all three ```video_mode``` settings

**Comment by prathibhacdac** [comment]
> To do what? explain further.
To prevent the user from calling their parents.
 
> Are you perhaps not looking for the Asterisk ConfBridge solution?

**Comment by prathibhacdac** [comment]
> Yes, its mostly implemented, I still have to complete the inter-conference message sending, that a caller can send a text message to all participants of the conference call.
>
When will this be released?

**Comment by InnovateAsterisk** [comment]
It’s been released for a while now. 

Make sure you create a buddy as type “extension”. So that you have video option.

**Comment by prathibhacdac** [comment]
There is no provision to give the buddy type.

**Comment by InnovateAsterisk** [comment]
To Dial into a conference you would have made some sort of feature code or even a DID. eg *88. or 1102 that would take you into the conference room. When you make the contact/buddy make sure you click the add buddy icon, and leave it on "Basic Extension" and dont change it to "Address Book Contact".

![image](https://user-images.githubusercontent.com/58937573/162677784-b1ffefe5-77c2-4441-958e-8fed2ce22bd8.png)
